common/src/java/com/zimbra/common/util/StringUtil.java

```diff
@@ -889,7 +889,7 @@ public static String escapeJavaIdentifier(String s) {
         for (int i = 0; i < s.length(); i++) {
             char ch = s.charAt(i);
             if (i == 0) {
-                result.append(Character.isJavaIdentifierStart(ch) ? ch : "_");
+                result.append(Character.isJavaIdentifierStart(ch) ? ch : "_" + ch);
```

Comment:
escapeJavaIdentifier can you please check references and see if there are any other places this utility function used.  look from github in zimbra org.

Reply:
had checked it, its being only used here.